packages/widgets/src/accordionpanel.ts

```diff
@@ -296,6 +306,26 @@ export class AccordionPanel extends SplitPanel {
     }
   }
 
+  private _toggleExpansion(index: number) {
+    const title = this.titles[index];
+    const widget = (this.layout as AccordionLayout).widgets[index];
```

Comment:
Presumably since you can control when this method is invoked it's not necessary to guard, here.

Reply:
Agreed so I will not add guard here.